### DISCUSSION TITLE
posts/git-merge-intro/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#25] [Announcements]
# posts/git-merge-intro/

合并在 Git 中是一个十分常见的操作：整合不同分支之间的更改，或者对远程分支执行 pull 及 push 操作，都需要进行合并。 但对新手来说， git merge 这一命令……

https://waynerv.com/posts/git-merge-intro/

**Comment by ZJUGuoShuai** [comment]
好文章，读完有收获，感谢 :heart:

**Comment by yanhaijing** [comment]
支持下

**Comment by chengtianbao** [comment]
文章写的太好了,看了受益匪浅.请问文章的GIF图片是用什么软件制作的呢👀👀

**Reply by waynerv** [reply]
动画模拟：https://git-school.github.io/visualizing-git/
录制成GIF：GIPHY CAPTURE

**Comment by DarrenMan** [comment]
清晰了  好文 👍🏻

**Comment by Niko-Laputa** [comment]
感谢感谢，受教了
